permissions_engine/fetch_keys.py

```diff
@@ -0,0 +1,22 @@
+import requests
+import json
+issuers = ["http://localhost:8080/auth/realms/mockrealm"]
+
+data = dict()
+data["keys"] = dict()
+data["keys"]["https://oidc:8443/auth/realms/mockrealm"] = requests.get("http://localhost:8080/auth/realms/mockrealm/protocol/openid-connect/certs").text
+
+
+
+#
+# issuer and where certs get fetched from do not match 
+# can't use solution below
+#
+# for i in issuers:
+#     response = requests.get(i + "/.well-known/openid-configuration")
+#     json_data = json.loads(response.text)
+#     jwks = json.loads(requests.get(json_data["jwks_uri"]).text)
+#     data["keys"][i] = jwks
+
+with open('permissions_engine/data.json', 'w') as f:
```

Comment:
Let's call this something other than data - data is a little confusing because `data` is also implicitly the top-level directory for all the rego files  (e.g. in permissions rego, the lines:

```
import data.idp.valid_token
import data.idp.trusted_researcher
import data.idp.username
```

How about keys or idp_configuration or...

Reply:
I can't really change the file name to something else. I think rego only allows either data.json or input.json. Here is the error I got from changing name to "data_keys.json"
```
error: load error: 1 error occurred during loading: /idp.rego:7: rego_parse_error: unexpected import path, must begin with one of: {data, input}, got: data_keys

import data_keys

```

Reply:
Ok, fair enough.  Let's leave it the way it is then for now, but I think in the longer term it would be good to save the data as a .rego file (so we could add other things to it too).